rudder-core/src/main/resources/hooks.d/policy-generation-finished/readme.txt

```diff
@@ -0,0 +1,39 @@
+
+Generallity about Rudder Hooks
+==============================
+
+Hooks must be executable file, typically scripts, whatever the language
+as long they are self-executable (i.e: correct #! definition). 
+
+Hooks must be name with the extension ".hook". Any other extension will 
+be ignore (which allow to put other file in these directory, like a 
+readme, for example). 
+
+Hooks are executed sequentially, in alpha-numeric order. We encourage
+you to use the patter "NN-hookname.ext.hook", with NN a number like
+"05", "20", etc. 
+
+About policy-generation-finished/ hooks
+==================================
+
+When/What ?
+-----------
+
+This directory contains hooks executed after poplicies are fully 
+generated for all nodes, and these new policies are available for
+download for the node. 
+
+Typically, these hooks are used to log information about the
+generation which just happened or notify third parties that
+new policies are available (for ex: cf-serverd SIGHUP) 
```

Comment:
Post hook generation are executed at each generation, even if no nodes are modified. Always possible to opt-out by testing for empty list of nodes.